src/NuGet.Core/NuGet.Common/Telemetry/NuGetTelemetryService.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+
+namespace NuGet.Common
+{
+    /// <summary>
+    /// Base class for nuget telemetry services
```

Comment:
I was thinking to have a common place to emit telemetry event... But after a second thought, I think you're right we dont actually need this and specific telemetry service class will be able to just call into telemetry session and pass telemetry event.